Doc/library/fcntl.rst

```diff
@@ -176,17 +176,28 @@ The module defines the following functions:
    method are accepted as well) of the file to lock or unlock, and *cmd*
    is one of the following values:
 
-   * :const:`LOCK_UN` -- unlock
-   * :const:`LOCK_SH` -- acquire a shared lock
-   * :const:`LOCK_EX` -- acquire an exclusive lock
+   .. data:: LOCK_UN
 
-   When *cmd* is :const:`LOCK_SH` or :const:`LOCK_EX`, it can also be
-   bitwise ORed with :const:`LOCK_NB` to avoid blocking on lock acquisition.
-   If :const:`LOCK_NB` is used and the lock cannot be acquired, an
+             release an existing lock
+
+   .. data:: LOCK_SH
+
+             acquire a shared lock
+
+   .. data:: LOCK_EX
+
+             acquire an exclusive lock
+
+   .. data:: LOCK_NB
+
+             bitwise OR with any of the other three constantsoperations to make
+             the request non-blocking.
```

Comment:
```suggestion
      Bitwise OR with any of the other three ``LOCK_*`` constants to make
      the request non-blocking.
```

* Fix apparent typo
* Make it fully explicit which other constants are being referred to
* Fix formatting per Style Guide and standard convention, as above